tensorflow/core/kernels/mkl/mkl_slice_op.cc

```diff
@@ -325,7 +323,7 @@ class MklSliceOp : public OpKernel {
 
     if (!context->status().ok() || done == true) return;
 
-    // Though MKL-DNN supports more than 8 dimension and
+    // Though oneDNN supports more than 8 dimension and
     // less than 12 dimension tensor.
```

Comment:
Nit: 
```suggestion
    // oneDNN supports tensors with 8-12 dimensions.
```

Reply:
Fixed and merged to comment lines into one. Commit will come after local build test.